### ISSUE TITLE
Error page visible

### ISSUE THREAD

**Issue by ghost** [kirby-deprecated-plugins/kirby-panel-prevnext#1]
Hi Jens,
Thanks for this plugin, I found it very useful ;)
I have excluded the **error page** in the panel with: `hide: true`. There is no reason that the user edit and change this page.
But, navigating back and forth between pages, with this plugin, the error page is still visible and accessible, how can I remove it?

**Comment by jenstornell** [comment]
I agree that I can be a good idea for the plugin to respect the `hide: true`. I'll keep this issue open.

**Comment by uniqeorn** [comment]
you can have it all right now and you can do something about yourself
